tests/ci/sync-to-aws.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -euo pipefail
+
+source /tmp/.env
+
+[ -z "$IMAGE_KEY" ] && error "Could not find '$IMAGE_KEY', qcow image has not been created"
+DOWNLOAD_DIRECTORY="/var/lib/libvirt/images"
+S3_BUCKET_NAME="fedora-testing-farm-image-import"
+AWS_CLI="aws"
+
+# Convert qcow2 image to raw image
+echo "[+] Converting from qcow2 to raw format"
+qemu-img convert -p ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.qcow2 ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.raw
+
+# Upload the raw and qcow2 images to s3 bucket
+echo "[+] Uploading raw image to 's3://${S3_BUCKET_NAME}/${IMAGE_KEY}.raw"
+$AWS_CLI s3 cp ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.raw s3://${S3_BUCKET_NAME}
+$AWS_CLI s3 cp ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.qcow2 s3://${S3_BUCKET_NAME}
+
+# Remove the downloaded files
+echo "[+] Removing downloaded and converted image"
+rm -f ${DOWNLOAD_DIRECTORY}/${IMAGE_KEY}.{qcow2,raw}
```

Comment:
I think that this part should be done at the `clean-up.sh` script. Actually, it already deletes the `qcow2` file.
The good thing about doing it there is that it's called at the `finish` stage from `tmt`, which makes sure it always get called, even if something fails in a previous steps.
Right now, if something fails before, the file would never gets removed.

Reply:
Done